examples/metajob-example/example-meta-job/example_meta_job.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from vdk.plugin.meta_jobs.meta_job_runner import MetaJobInput
+
+
+JOBS_RUN_ORDER = [
+    {
+        "job_name": "ingest-job1",
+        "team_name": "my-team",
+        "fail_meta_job_on_error": True,
+        "depends_on": [],
+    },
+    {
+        "job_name": "ingest-job2",
+        "team_name": "my-team",
+        "fail_meta_job_on_error": True,
+        "depends_on": [],
+    },
+    {
+        "job_name": "read-data-job",
+        "team_name": "my-team",
+        "fail_meta_job_on_error": True,
+        "depends_on": ["ingest-job1", "ingest-job2"],
+    },
+]
+
+
+def run(job_input):
+    MetaJobInput().run_meta_job(JOBS_RUN_ORDER)
```

Comment:
Just an idea. I think it would be easier for user if the paramethers are implemented is such way that can be used like:
```
some_object.run_meta_job(
    job_name = "ingest-job1",
    team_name = "my-team",
    fail_meta_job_on_error = True,
    depends_on = []
)

some_object.run_meta_job(
  **{
      "job_name": "read-data-job",
      "team_name": "my-team",
      "fail_meta_job_on_error": True,
      "depends_on": ["ingest-job1", "ingest-job2"]
  }
)
```
I like what I currently see. This is just a suggestion.

Reply:
will consider this, but it is not in the scope of this PR

Reply:
There is a ticket about that idea here - https://github.com/vmware/versatile-data-kit/issues/1271 
Feel free to follow it /upvote it @victor-partico  :)